daml-lf/spec/daml-lf-1.rst

```diff
@@ -2785,6 +2797,12 @@ as described by the ledger model::
    —————————————————————————————————————————————————————————————————————— EvUpdExercWithoutActors
      'exercise_without_actors' Mod:T.Ch cid v₁ ‖ (st₀, keys₀)  ⇓ᵤ  ur
 
+     cid ∉ dom(st)
+   —————————————————————————————————————————————————————————————————————— EvUpdFetchMissing
+     'fetch' @Mod:T cid ‖ (st; keys)
+       ⇓ᵤ
+     Err "Exercise on unknown contract"
```

Comment:
Shouldn't that refer to `fetch` rather than `exercise`?
```suggestion
     Err "Fetch of unknown contract"
```

Reply:
No, it's not a mistake. Fetch errors look like exercise errors (see the other fetch error case). I don't know if this is desirable, but it's closer to the implementation. My guess is that we'll have to redo all the errors at some point (looking at the future w/ exception handling) and we can fix this then.